submission/programs/adadas.r

```diff
@@ -27,17 +25,20 @@ library(stringr)
 library(xportr)
 library(pilot3)
 
-dm <- haven::read_xpt(file.path(path$sdtm, "dm.xpt"))
-qs <- haven::read_xpt(file.path(path$sdtm, "qs.xpt"))
-adsl <- haven::read_xpt(file.path(path$adam, "adsl.xpt"))
+sdtm_path <- "./submission/sdtm/"
+adam_path <- "./submission/adam/"
```

Comment:
I just want to ensure that these paths won't stay hard coded in each of the adam programs.   

if needed you can run this first : 
```
path <- list(
  sdtm = "./submission/sdtm",   # Modify path to the sdtm location
  adam = "./submission/adam",    # Modify path to the adam location
  output = "./submission/output"    # Modify path to the output location
)


```

then you should be able to run the prior version using `path$sdtm` or `path$adam`.

Reply:
sure we can do that, but it still hardcoded - just in a list.  I find accessing a list with `$` to be slightly confusing for new folks.    Happy to change it if that is what you want to do

Reply:
Thanks, @bms63 .  Changing it back will be consistent with Pilot 1 as we updated in #116 and #117 .